.travis.yml

```diff
@@ -30,6 +30,7 @@ jobs:
         # only linux supports docker https://docs.travis-ci.com/user/docker/ => tests with docker skipped programmatically ~ OS
         - docker-compose up -d
         - make test-coverage codecov
+        - make test-race
```

Comment:
why using `make ci-script` instead?
https://github.com/src-d/ci/blob/v1/Makefile.main#L278

Reply:
@dpordomingo I forgot that my PR to `ci` was merged already xD thank you!

Reply:
Done

Reply:
thanks for PR this!